web/handlers/admin/allow_list.go

```diff
@@ -64,6 +64,10 @@ func (h allowListH) overview(rw http.ResponseWriter, req *http.Request) (interfa
 	if err != nil {
 		return nil, err
 	}
+	// Reverse the slice to provide recent-to-oldest results
+	for i, j := 0, len(lst)-1; i < j; i, j = i+1, j-1 {
+		lst[i], lst[j] = lst[j], lst[i]
+	}
```

Comment:
@cryptix What do you think about this?

Reply:
yup, nice use of the multi assign.

If you would want to sort by something other then the index of the array, I'd say go for [sort.Reverse](https://golang.org/pkg/sort/#Reverse) but this is fine until we change `List(ctx)` to have options and let the database do the work.

Reply:
Oh, didn't know about `sort.Reverse`. Yeah, and I'll revisit this code once I work on pagination

Reply:
you would have to implement sort.Interface, like [here](https://github.com/cryptoscope/ssb/blob/f8b517c21b5d9b8fb05f923f29e59dda9a1730e5/sbot/status.go#L66).